soccer/gameplay/plays/restarts/basic_direct.py

```diff
@@ -0,0 +1,49 @@
+import robocup
+import constants
+import standard_play
+import enum
+import behavior
+import main
+import evaluation
+import tactics.coordinated_pass
+import skills
+
+
+class DirectFreeKick(standard_play.StandardPlay):
+
+    class State(enum.Enum):
+        kicking = 1
+        passing = 2
+    def __init__ (self):
+        super().__init__(continuous = True)
+
+        self.add_state(DirectFreeKick.State.kicking,behavior.Behavior.State.running)
+        self.add_state(DirectFreeKick.State.passing,behavior.Behavior.State.running)
```

Comment:
This can be condensed into a for loop.

```
        for s in DirectFreeKick.State:
            self.add_state(s, behavior.Behavior.State.running)
```